finalPrep/README.md

```diff
@@ -44,31 +43,29 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
-
+- [Week 1](https://user-images.githubusercontent.com/99286590/156654509-97c0e1a3-011b-4630-b588-b68f9f8ee79e.png)
+- [Week 2](https://user-images.githubusercontent.com/99286590/156654515-34c292de-99cf-4844-9ad7-a2db0f07838e.png)
+- [Week 3](https://user-images.githubusercontent.com/99286590/156654523-1da2ca0a-bc99-48c1-9d3e-a7d267b7b019.png)
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
 - [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
-
+  - [ ] Add link to your gist here: (https://gist.github.com/willhobson85/25624baa1dfc0189d51aff24c1d68340.js)
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
 - [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+  - [ ] Add a link to your gist here: (https://gist.github.com/willhobson85/233d1e6970f28a35200bb41233d66d5f.js)
 
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
 - [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+  - [ ] Add a link to your gist here: (https://gist.github.com/willhobson85/b5bb72ec83ab1d595568368c064fed11.js)
 
```

Comment:
All these prep gists look good - just an FYI, the URLs all end with `.js` and that takes one to a giant JavaScript file - I removed that extension and was able to see your gists. Letting you know because you should be coming back to  the Group Work and Mentor DTRs and I don't want you to think you lost them.